api-admin/src/main/java/bio/terra/pearl/api/admin/service/auth/AuthUtilService.java

```diff
@@ -25,6 +25,13 @@
 /** Utility service for common auth-related methods */
 @Service
 public class AuthUtilService {
+  /**
+   * 'BASE_PERMISSON' indicates the operation is permitted for any PortalAdminUser as they are
+   * authorized to access the given portal. It might include public-ish operations like viewing
+   * surveys, etc.
+   */
+  public static final String BASE_PERMISSON = "BASE";
```

Comment:
For my own understanding: how does the idea of a "base" permission differ from a "read"/"view" permission? Is the idea that there will be other non-read actions that we'll want to capture with the base permission without having to enumerate every piece of functionally as a new permission, and so calling it "read" wouldn't be accurate?

Reply:
I think there are reads that would not be included in the BASE permission.  For example, reading participant data shouldn't be in "BASE"--that should require a read_participant_data.  And there might be trivial writes that don't require a permission (e.g. updating your own user preferences)

Reply:
Got it- makes sense! I appreciate the clarification